config/Sage/game_options.txt

```diff
@@ -625,6 +625,9 @@ ALLOW_CREW_OBJECTIVES
 ## Uncomment to restrict suiciding. Adds an additional confirmation dialogue, additional logging, and prevents suicide within the first 15 minutes of the round.
 RESTRICTED_SUICIDE
 
+## Uncomment to make donator cosmetics donator-only, leaving this commented will give everyone free access to all donator items
```

Comment:
Was there another decision made? Unless we are reversing our decision, this comment should be updated.